salt/states/pip_state.py

```diff
@@ -440,9 +440,28 @@ def installed(
             - require:
               - pkg: python-pip
 
-    This will install the latest Django version greater than 1.6 but less
+    Installs the latest Django version greater than 1.6 but less
     than 1.7.
 
+    pkgs
+        A list of python packages to install. This let you install multiple
+        packages at the same time.
+
+    Example:
+
+    .. code-block:: yaml
+
+        django-and-psycopg2:
+          pip.installed:
+            - pkgs:
+              - django >= 1.6, <= 1.7
+              - psycopg2 >= 2.8.4
+            - require:
+              - pkg: python-pip
+
+    Installs the latest Django version greater than 1.6 but less than 1.7
+    and the latest psycopg2 greater than 2.8.4 at the same time.
+
```

Comment:
Same thing here, all this should be indented under the `pkgs` argument.

Reply:
I only reproduced what was previously done, I though this was the current documentation style